app/controllers/general.php

```diff
@@ -350,13 +351,26 @@ function router(App $utopia, Database $dbForConsole, callable $getProjectDB, Swo
         $utopia->getRoute()?->label('error', '');
         return false;
     } else {
-        throw new AppwriteException(AppwriteException::GENERAL_SERVER_ERROR, 'Unknown resource type ' . $type);
+        throw new AppwriteException(AppwriteException::GENERAL_ACCESS_FORBIDDEN, 'Unknown resource type ' . $type);
```

Comment:
What's the reasoning for this change?

Reply:
Copied from this PR
https://github.com/appwrite/appwrite/pull/7954/files#diff-ff12d23a7b6584b45f7e669b24ae1637106903cbac9390cad416e2c21dfda3f4R375

Reply:
Wrong copy fixed now